app/assets/stylesheets/components/_banner.scss

```diff
@@ -141,4 +141,29 @@ $line-color: $primary;
     animation-iteration-count: infinite;
     animation-timing-function: cubic-bezier(0.1, 0.7, 1.0, 0.1);
   }
+
+  .banner-leads-form {
+    position: absolute;
+    left: 50%;
+    bottom: 75px;
+    transform: translateX(-50%);
+    z-index: 5;
+  }
```

Comment:
I tried breaking it by reducing the height and didn't manage to. Clean CSS.

It does break on phones tho. But it's not our priority and the current style is better overall, so I'm just making a note of this and moving on. Don't worry about it unless you find a quick fix. (I'd start with just hiding Matt's caption).
![image](https://user-images.githubusercontent.com/1820441/111186381-95291780-8578-11eb-8d36-5a69fc479005.png)